client/scripts/views/pages/new_snapshot_proposal/new_proposal_form.ts

```diff
@@ -105,14 +92,14 @@ const newThread = async (
       : JSON.stringify(quillEditorState.editor.getContents());
 
   form.body = bodyText;
-  form.snapshot = await getBlockNumber(getProvider(space.network));
+  form.snapshot = await snapshotJs.utils.getBlockNumber(snapshotJs.utils.getProvider(space.network));
   form.metadata.network = space.network;
   form.metadata.strategies = space.strategies;
 
   const msg: any = {
     address: author.address,
     msg: JSON.stringify({
-      version,
+      version: '0.1.3',
```

Comment:
why do we need to specify the version here? can we fetch the appropriate version from elsewhere?